### DISCUSSION TITLE
JSON TO SOAPXML NODE 18 TYPESCRIPT

### DISCUSSION THREAD

**Discussion by maloybain** [npm/feedback#988] [General]
Hello,
I do not see any way by which I can feed a json to a library which will create a soap xml.
xml2js or fast-xml-parser will not help as we want an soap xml.
In java world we can create client classes from xsd. then use API to generate soap xml by populating values in the client classes.
I am using node 18 typescript.

Kindly advise and help

**Comment by MylesBorins** [comment]
We do not offer general support through this feedback channel
